src/ini.pest

```diff
@@ -1,16 +1,22 @@
-ini = { SOI ~ (pair | comment | NEWLINE)* ~ section* ~ EOI }
-WHITESPACE = _{ " " | "\t" }
+ini = { SOI ~ (pair | comment)* ~ section* }
+
+// lines
+pair = { key ~ "=" ~ value ~ eol}
```

Comment:
shouldn't the value be optional?

Reply:
That's a really good question. I haven't seen any specification yet allowing explicitly empty values and that would probably also make no sense. Since if a value is empty we could also say it does not exist, right?

I put that on the list of features to examine.

Reply:
I'm inclined to agree; though, JSON, e.g., has the ability to have a key without a `null` value? The result of `JSON.stringify({ foo: undefined })`, however, is just `{}`.

Reply:
I think we should remove this silly feature!